src/opm/io/eclipse/ESmry.cpp

```diff
@@ -28,6 +28,7 @@
 #include <limits>
 #include <limits.h>
 #include <set>
+#include "boost/filesystem.hpp" 
```

Comment:
The "tradition" in the codebase is to use `<...>` for installed headers.

Reply:
Sorry, fixed